### PR TITLE
Gauntlet stMATIC and MaticX Update

### PR DESCRIPTION
**Pull request by reem**

### CODE REVIEW

**Review by sakulstra — DISMISSED** [review]

File: src/20231117_AaveV3Polygon_GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2/GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Gauntlet has not received any compensation from any third-party in exchange for recommending any of the actions contained in this proposal.
+
+By approving this proposal, you agree that any services provided by Gauntlet shall be governed by the terms of service available at gauntlet.network/tos.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+_By approving this proposal, you agree that any services provided by Gauntlet shall be governed by the terms of service available at gauntlet.network/tos._
```
Comment (sendra) [inline src/20231117_AaveV3Polygon_GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2/GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2.md:39]: Isn't this already specified in the Disclaimer? Does it need to be also on Copyright?

Comment (reem) [reply src/20231117_AaveV3Polygon_GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2/GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2.md]: Thanks for catching, removed the duplication

**Comment by reem** [comment]
Added deployed payload address

File: src/20231117_AaveV3Polygon_GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2/GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2_20231117.s.sol [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    // compose actions for validation
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsPolygon = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsPolygon[0] = GovV3Helpers.buildAction(0x54dC357F7461BcEEE5BdbA80996f5CB7d7512445);
+    payloads[0] = GovV3Helpers.buildPolygonPayload(vm, actionsPolygon);
+
+    // create proposal
+    vm.startBroadcast();
+    GovV3Helpers.createProposal2_5(
```
Comment (sakulstra) [inline src/20231117_AaveV3Polygon_GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2/GauntletRecommendationToLowerStMATICMaticXNonEmodeLTPt2_20231117.s.sol:44]:
```suggestion
    GovV3Helpers.createProposal2_5(
      vm,
```

**Comment by reem** [comment]
Accepted changes, merged main, re-ran tests (no changes generated)

**Review by sakulstra — APPROVED** [review]

**Pull request merged**